tests/test_web_runner.py

```diff
@@ -265,3 +265,29 @@ async def shutdown():
 
     web.run_app(app)
     assert spy.called, "run_app() should work after asyncio.run()."
+
+
+async def test_app_runner_serve_forever_uninitialized(make_runner) -> None:
+    runner = make_runner()
+    with pytest.raises(RuntimeError):
+        await runner.serve_forever()
+
+
+async def test_app_runner_serve_forever_concurrent_call(make_runner, loop) -> None:
+    runner = make_runner()
+    task = loop.create_task(runner.serve_forever())
+    await asyncio.sleep(0.01)
+    with pytest.raises(RuntimeError):
```

Comment:
Can we use the match argument to match the text of the exception in these?
The codecov comment suggests this test isn't working correctly.